src/routes/raids/ViewRaid.tsx

```diff
@@ -2,12 +2,68 @@ import React, { useEffect, useState } from 'react'
 import firestore from '#utils/firebase'
 import { useParams } from 'react-router-dom'
 
+import styled from '@emotion/styled'
+
 import UserStatBlock from '#components/user-stat-block'
 import Emoji from '#components/emoji'
 import LoadingSpinner from '#components/loading-spinner'
 
 type State = 'loading' | 'success' | 'not-found' | 'error'
 
+const StatsView = styled.main`
+  height: 100vh;
+  display: grid;
+  grid-template-rows: auto 1fr;
+  > * {
+    padding: 1rem;
+  }
+`
+
+const Header = styled.header`
+  z-index: 1;
+  background-color: white;
+  border-bottom: 1px solid hsl(0, 0%, 78%);
+  box-shadow: 0 2px 4px 0 hsla(0, 0%, 0%, 10%);
+  > div {
+    margin: 0 auto;
+    max-width: 700px;
+  }
+
+  > div > h1 {
+    font-size: 2rem;
+    line-height: 3rem;
+    margin-bottom: 2rem;
+  }
+`
+
+const TotalStats = styled.div`
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+
+  > p {
```

Comment:
Is there a chance we can create a styled component out of this and not have inlined styles? I know this was initially there, but I would propose creating a styled component and giving it a proper name :+1:.

Reply:
Has been discussed here: https://github.com/OpenSourceRaidGuild/website/issues/24

Reply:
My concern with this is that we end up with a lot of components like *Container